src/RapidezStatamicServiceProvider.php

```diff
@@ -63,16 +82,17 @@ public function bootComposers() : self
             });
         }
 
-        View::composer('rapidez::layouts.app', function ($view) {
-            foreach (GlobalSet::all() as $set) {
-                foreach ($set->localizations() as $locale => $variables) {
-                    if ($locale == Site::current()->handle()) {
-                        $data[$set->handle()] = $variables;
-                    }
-                }
-            }
+        View::composer('rapidez::*', function ($view) {
```

Comment:
I think we need globals everywhere, not only in the rapidez namespace. This will only be available in `resources/views/vendor/rapidez` but new views in a project won't be added here.

Reply:
I had it as * originally, but it turns out the statamic globals page breaks with that because it happens to conflict with its own variable called $globals. And there doesn't seem to be a clean way to exclude specific views from a composer.

Perhaps changing the variable name would be a solution but that'd be a breaking change.